### PR TITLE
JBIDE-28548: OpenShift Application explorer RH Developer Sandbox logi…

### PR DESCRIPTION
**Pull request by jeffmaury**
…n screen layout is broken on windows 10

Signed-off-by: Jeff MAURY <jmaury@redhat.com>

### CODE REVIEW

**Review by sbouchet — COMMENTED** [review]
LGTM on linux, but need confirmation on win10.

**Review by odockal — APPROVED** [review]

**Comment by odockal** [comment]
@jeffmaury I was not able to verify on windows, too many broken deps when trying to build it locally in eclipse workspace. We need to bump up, otherwise LGTM. Will test once it is merge+build.

**Pull request merged**
